pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/ZookeeperResource.java

```diff
@@ -256,6 +257,56 @@ public SuccessResponse putData(
     }
   }
 
+  @POST
+  @Path("/zk/create")
+  @Authorize(targetType = TargetType.CLUSTER, action = Actions.Cluster.UPDATE_ZNODE)
+  @Authenticate(AccessType.CREATE)
+  @Produces(MediaType.APPLICATION_JSON)
+  @Consumes(MediaType.APPLICATION_JSON)
+  @ApiOperation(value = "Create a node at a given path")
+  @ApiResponses(value = {
+      @ApiResponse(code = 200, message = "Success"), @ApiResponse(code = 204, message = "No Content"),
+      @ApiResponse(code = 400, message = "Bad Request"), @ApiResponse(code = 500, message = "Internal server error")
+  })
+  public SuccessResponse createNode(
```

Comment:
This is pretty much the same as putData API, except ttl option.
Should you add a new parameter like `force` default to `true` into `putData` API instead of creating a new one?
You can add a new exception into it like `NodeAlreadyExist`.

Reply:
This complicates the existing API. Now we have 1. `expectedVersion` which is not required for creating new node. 2. `ttl`  which is not required to update a new node. Not to mention the POST vs PUT semantics would also be incorrect.  Wdyt?